### ISSUE TITLE
Improve adding keycloak plugins

### ISSUE THREAD

**Issue by filippomc** [MetaCell/cloud-harness#555]
- [ ] Copy all plugins folder in the accounts Dockerfile like `COPY plugins/ /opt/jboss/keycloak/standalone/deployments/`
- [ ] Add an option to enable plugins from the account's values.yaml file instead of overriding the full realm.json file.

application/accounts/values.yaml
```yaml
plugins:
- metacell-admin-event-listener
- my-plugin
```

**Comment by zsinnema** [comment]
@filippomc for me this is a very specific requirement (only for Keycloak). May be it's better to create some kind of "hook" mechanism to execute some "script(s)" after/during build / deploy / init

for this we could e.g. execute a script that enables the plugin

what do you think?

**Comment by filippomc** [comment]
not sure we need a script for this as plugins can be specified in the realm.json file.
But I also see that the proposal above may be too naive as depending on the plugin there is a different configuration point.

For example for event listeners we need just the plugin name like:

```json
 "eventsListeners": [
            "metacell-admin-event-listener",
            "jboss-logging"
        ],
```

While other plugins may need a specific configuration. For example identity provider:
```json
"identityProviders": [
            {
              "alias": "orcid",
              "internalId": "bcc1bc98-c0e5-4f95-8acf-f679e3abe156",
              "providerId": "orcid",
              "enabled": true,
              "updateProfileFirstLoginMode": "on",
              "trustEmail": false,
              "storeToken": false,
              "addReadTokenRoleOnCreate": false,
              "authenticateByDefault": false,
              "linkOnly": false,
              "firstBrokerLoginFlowAlias": "first broker login",
              "config": {
                "syncMode": "IMPORT",
                "clientSecret": "**********",
                "clientId": "*******",
                "useJwksUrl": "true"
              }
            }
          ],
```

**Comment by zsinnema** [comment]
@filippomc I was reffering to the fact that this is a very specific feature for Keycloak . Can we come up with a more "generic" solution. I was trying to do so by adding some kind of a hook that you then can use to do application specific stuff

**Comment by zsinnema** [comment]
@filippomc after our chat, please ignore my last comment  :-)

**Comment by filippomc** [comment]
Agreed that as a first step we add the metacell-admin-event-listener by default in the realm.json if the events application is deployed
